src/edwards.rs

```diff
@@ -647,17 +647,13 @@ impl EdwardsBasepointTable {
 
 impl ExtendedPoint {
     /// Multiply by the cofactor: compute `8 * self`.
-    ///
-    /// Convenience wrapper around `mult_by_pow_2`.
-    #[inline]
```

Comment:
why are we not inlining this?

Reply:
just because I would rather not go down the road of "mess with llvm inlining cost model" unless necessary